src/turnkeyml/cli/cli.py

```diff
@@ -107,7 +102,9 @@ def check_extension(choices, file_name):
         "input_files",
         nargs="+",
         help="One or more script (.py), ONNX (.onnx), or input list (.txt) files to be benchmarked",
-        type=lambda file: check_extension(("py", "onnx", "txt"), file),
+        type=lambda file: check_extension(
+            ("py", "onnx", "txt"), file, benchmark_parser.error
```

Comment:
`benchmark_parser.error` is the standard argparse error-throwing function for the `benchmark` command.